### PR TITLE
Added VideoEncoding files

### PR DESCRIPTION
**Pull request by IngridAtMicrosoft**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: VideoEncoding/CreateTransform_Job_PresetOverride/create-transform-job-preset-override.py [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+# Set the attributes of the input Asset using the random number
+in_asset_name = 'inputassetName' + uniqueness
+in_alternate_id = 'inputALTid' + uniqueness
+in_description = 'inputdescription' + uniqueness
+
+# Create an Asset object
+# From the SDK
+# Asset(*, alternate_id: str = None, description: str = None, container: str = None, storage_account_name: str = None, **kwargs) -> None
+# The asset_id will be used for the container parameter for the storage SDK after the asset is created by the AMS client.
+input_asset = Asset(alternate_id=in_alternate_id,description=in_description)
```
Comment (iscai-msft) [inline VideoEncoding/CreateTransform_Job_PresetOverride/create-transform-job-preset-override.py:71]: nit, space after comma

File: VideoEncoding/CreateTransform_Job_PresetOverride/create-transform-job-preset-override.py [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+
+# Set the attributes of the output Asset using the random number
+out_asset_name = 'outputassetName' + uniqueness
+out_alternate_id = 'outputALTid' + uniqueness
+out_description = 'outputdescription' + uniqueness
+# From the SDK
+# Asset(*, alternate_id: str = None, description: str = None, container: str = None, storage_account_name: str = None, **kwargs) -> None
```
Comment (iscai-msft) [inline VideoEncoding/CreateTransform_Job_PresetOverride/create-transform-job-preset-override.py:78]: there's a lot of commented out code you should remove

File: VideoEncoding/CreateTransform_Job_PresetOverride/create-transform-job-preset-override.py [hunk 3]
```diff
@@ -0,0 +1,359 @@
...
+
+# The AMS Client
+print("Creating AMS Client")
+client = AzureMediaServices(default_credential, SUBSCRIPTION_ID)
+
+# Create an input Asset
+print(f"Creating input asset {in_asset_name}")
+# From SDK
+# create_or_update(resource_group_name, account_name, asset_name, parameters, custom_headers=None, raw=False, **operation_config)
+inputAsset = client.assets.create_or_update( RESOURCE_GROUP, ACCOUNT_NAME, in_asset_name, input_asset)
```
Comment (iscai-msft) [inline VideoEncoding/CreateTransform_Job_PresetOverride/create-transform-job-preset-override.py:89]: space before RESOURCE_GROUP

File: VideoEncoding/CreateTransform_Job_PresetOverride/create-transform-job-preset-override.py [hunk 4]
```diff
@@ -0,0 +1,359 @@
...
+      print(job_current2.state)
+      # TODO: Provide Error details from Job through API
+      return
+    else:
+      print(job_current2.state)
+      countdown(int(time_in_seconds))
+
+time_in_seconds = 10
+countdown(int(time_in_seconds))
+#</CheckJob>
```
Comment (iscai-msft) [inline VideoEncoding/CreateTransform_Job_PresetOverride/create-transform-job-preset-override.py:359]: there are some html like comments in the samples, are they needed?

**Review by iscai-msft — COMMENTED** [review]
two things: there's a lot of comments and commented out code in these samples. For user experience and general readability, I think it might be good to have less text thrown at the user.

Also are there async samples?

**Comment by sunzae99** [comment]
> two things: there's a lot of comments and commented out code in these samples. For user experience and general readability, I think it might be good to have less text thrown at the user.
> 
> Also are there async samples?

I will take a look at them and remove the comments that are not necessary.
Also, the async samples have been included for the Live Events, which I shall be creating a new PR for that .

File: VideoEncoding/CreateTransform_Job_PresetOverride/create-transform-job-preset-override.py [hunk 5]
```diff
@@ -0,0 +1,318 @@
...
+                    width = "1280",
+                    height = "720",
+                    b_frames = 4,
+                    label = "HD-1800kbps" # This label is used to modify the file name in the output formats
+                )
+            ]
+        ), 
+        AacAudio(
+            # Add an AAC audio layer for the audio encoding
+            channels = 2,
```
Comment (iscai-msft) [inline VideoEncoding/CreateTransform_Job_PresetOverride/create-transform-job-preset-override.py:229]: when passing in parameters by name, there shouldn't be any spaces around the equals sign. This should be `channels=2,` etc

File: VideoEncoding/CreateTransform_Job_PresetOverride/create-transform-job-preset-override.py [hunk 6]
```diff
@@ -0,0 +1,318 @@
...
+        time.sleep(1) 
+        t -= 1
+    job_current2 = client.jobs.get(RESOURCE_GROUP, ACCOUNT_NAME, transform_name, job_name_HEVC)
+    if(job_current2.state == "Finished"):
+      print(job_current2.state)
+      # TODO: Download the output file using blob storage SDK
+      return
+    if(job_current2.state == "Error"):
+      print(job_current2.state)
+      # TODO: Provide Error details from Job through API
```
Comment (iscai-msft) [inline VideoEncoding/CreateTransform_Job_PresetOverride/create-transform-job-preset-override.py:311]: you still have TODOs in the sample

File: VideoEncoding/CreateTransform_Job_PresetOverride/create-transform-job-preset-override.py (hunk 5 above)
Comment (sunzae99) [reply VideoEncoding/CreateTransform_Job_PresetOverride/create-transform-job-preset-override.py:229]: Thanks for pointing them out. I'll have the files changed.

**Pull request merged**